blueocean-core-js/src/js/services/DefaultSSEHandler.js

```diff
@@ -76,6 +76,16 @@ export class DefaultSSEHandler {
         if (event.job_ismultibranch && !event.blueocean_job_branch_name) {
             return;
         }
+        // don't care about indexing events
+        if (event.job_multibranch_indexing_status === 'INDEXING') {
+            return;
+        }
+        // If we have a queued item but the branch isn't present, we need to refresh the pager
+        // this happens, for example, when you create a new pipeline in a repo that did not have one
+        const pipeline = this.pipelineService.getPipeline(`/blue/rest/organizations/${event.jenkins_org}/pipelines/${event.blueocean_job_pipeline_name}/`);
```

Comment:
Is `event.blueocean_job_pipeline_name` the fully-qualified name of the pipeline, e.g. the `fullName`? If not, this will fail for nested pipelines. That might not matter today in creation use case, but if there's an easy way to ensure we're using the full name we should.

Reply:
hm, good question, I'll check and escape it or not

Reply:
good call, it is _not_ escaped